modules/fabric-project/outputs.tf

```diff
@@ -16,7 +16,7 @@
 
 output "project_id" {
   description = "Project id (depends on services)."
-  value       = google_project.project.project_id
+  value       = trimprefix(google_project.project.id, "projects/")
```

Comment:
Doesn't depends_on = [google_project. project] work?

Reply:
I'm not sure which depends_on you're asking:
1. if you're asking about depends_on here in output then no, changing 
```hcl
  depends_on  = [google_project_service.project_services]
```
to 
```hcl
  depends_on  = [google_project.project]
```
does not work. Terraform ignores this explicit dependency when it has the output value available immediately. This is especially the case when output value comes from input attribute. 

2. If you're asking whether explicit depends on module.project works e.g. in a datasource that is using this output, then yes, it's working. But please see my comment in main thread. 
```hcl
data "google_storage_project_service_account" "gcs_account" {
  project = module.project.project_id
  depends_on = [module.project]
}
```

Reply:
I see, I assumed depends_on may block on `google_project.project` since some of the attribs are computed.